lib/Doctrine/DBAL/Schema/Column.php

```diff
@@ -21,13 +20,13 @@ class Column extends AbstractAsset
     protected $_type;
 
     /** @var int|null */
-    protected $_length = null;
+    protected $_length;
 
     /** @var int */
-    protected $_precision = 10;
```

Comment:
Not sure if this should be changed

Reply:
As precision is only applicable for numerical columns, I think it does make sense to drop the default.

Reply:
Another solution could be moving this information to Type.

Reply:
This is a DTO which just carries what's given to it. It shouldn't imply any defaults. Whatever owns the definition (DB schema or the definition in the application) should define it explicitly.

I'll add it to the upgrade notes.

Reply:
Reverted from this patch.